examples/standards/fxc/FX-PI-01_ParisExample.xml

```diff
@@ -0,0 +1,2798 @@
+								</DistanceMatrixElement>
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z2+z3">
+									<Name>Noctilien zone 2 to zone 3</Name>
+									<Distance>2</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_2"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_3"/>
+								</DistanceMatrixElement>
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z2+z4">
+									<Name>Noctilien zone 2 to zone 4</Name>
+									<Distance>3</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_2"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_4"/>
+								</DistanceMatrixElement>
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z2+z5">
+									<Name>Noctilien zone 2 to zone 4</Name>
+									<Distance>4</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_2"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_5"/>
+								</DistanceMatrixElement>
+								<!--matrix zone 3 -->
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z3+z3">
+									<Name>Noctilien zone 3 to zone 3</Name>
+									<Distance>3</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_3"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_3"/>
+								</DistanceMatrixElement>
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z3+z4">
+									<Name>Noctilien zone 3 to zone 4</Name>
+									<Distance>3</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_3"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_4"/>
+								</DistanceMatrixElement>
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z3+z5">
+									<Name>Noctilien zone 3 to zone 4</Name>
+									<Distance>4</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_3"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_5"/>
+								</DistanceMatrixElement>
+								<!--matrix zone 4 -->
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z4+z4">
+									<Name>Noctilien zone 4 to zone 4</Name>
+									<Distance>3</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_4"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_4"/>
+								</DistanceMatrixElement>
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z4+z5">
+									<Name>Noctilien zone 4 to zone 4</Name>
+									<Distance>4</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_4"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_5"/>
+								</DistanceMatrixElement>
+								<!--matrix zone 5 -->
+								<DistanceMatrixElement version="any" id="tsln:noctilien_z5+z5">
+									<Name>Noctilien zone 5 to zone 5</Name>
+									<Distance>3</Distance>
+									<StartTariffZoneRef version="any" ref="tsln:zone_5"/>
+									<EndTariffZoneRef version="any" ref="tsln:zone_5"/>
+								</DistanceMatrixElement>
```

Comment:
What about the opposite directions: zone 5 to zone 4, zone 5 to zone 3, etc.? Or should the terms StartTariffZoneRef and EndTariffZoneRef not be taken too literally?

Reply:
I think we will merge, when ok. Probably @Aurige will update the example later on anyway.